recipe/meta.yaml

```
@@ -20,16 +24,20 @@ requirements:
     - cmake
     - make
     - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - noqt6                          # [build_variant == "qt5"]
+    - noqt5                          # [build_variant == "qt6"]
   host:
-    - qt-main
-    - zlib
-  run:
-    - qt-main
-    - zlib
+    - qt-main                        # [build_variant == "qt5"]
+    - qt6-main                       # [build_variant == "qt6"]
+    # Compatible with 1.2 and 1.3
+    - zlib =1.2                      # [build_variant == "qt5"]
+    - zlib =1.3                      # [build_variant == "qt6"]
```

Why are you building the qt5 variant against zlib 1.2? Everything built against 1.2 is compatible with 1.3 (and we've patched this retroactively), so I don't see why this is necessary. It should stay
```suggestion
    - zlib
```

---

I believe this helped with dependency resolution for things built with older versions of zlib 1.2 that had pins of `<1.3`. I will build locally and see if I can install packages with just zlib.